### PR TITLE
ref: add data migrations for pickle->json for models.options.option

### PR DESCRIPTION
**Pull request by asottile-sentry**

### CODE REVIEW

File: src/sentry/migrations/0450_pickle_to_json_sentry_option.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    Option = apps.get_model("sentry", "Option")
+
+    for obj in RangeQuerySetWrapperWithProgressBar(Option.objects.all()):
+        # load pickle, save json
+        obj.save(update_fields=["value"])
+
+
+class Migration(CheckedMigration):
+    # data migration: must be run out of band
+    is_dangerous = True
```
Comment (wedamija) [inline src/sentry/migrations/0450_pickle_to_json_sentry_option.py:19]: `Option` has < 1k rows and `ControlOption` has 0 rows, so you could mark these not dangerous and let them run as part of the deploy if you want to save ops some hassle

**Review by wedamija — APPROVED** [review]
Should we just consolidate these into the same migration?

File: src/sentry/migrations/0450_pickle_to_json_sentry_option.py (hunk 1 above)
Comment (asottile-sentry) [reply src/sentry/migrations/0450_pickle_to_json_sentry_option.py:19]: oops, well they're merged now so I guess I'll go through the hassle

is there some way I can check the row count on prod? I still need to do the following tables:
- sentry_organizationoptions
- sentry_projectoptions
- sentry_useroption

Comment (wedamija) [reply src/sentry/migrations/0450_pickle_to_json_sentry_option.py:19]: Replied via slack

**Pull request merged**